.gitignore

```diff
@@ -1,2 +1,18 @@
 node_modules
 test/expected
+
+logs
+*.log
+npm-debug.log*
+.eslintcache
+/coverage
+/dist
+/local
+/reports
+/node_modules
+.DS_Store
+Thumbs.db
+.idea
+.vscode
+*.sublime-project
+*.sublime-workspace
```

Comment:
Really? Is this stuff necessary? Should be always developer responsibility, not project.